cli/graphql.md

```diff
@@ -1169,6 +1187,126 @@ Here is a complete list of searchable operations per GraphQL type supported as o
 | Float | `ne`, `gt`, `lt`, `gte`, `lte`, `eq`, `range`      |
 | Boolean | `eq`, `ne`      |
 
+## AWS CloudFormation Template Parameters
+
+Much of the behavior of the GraphQL Transform logic is configured by passing arguments
+to the directives in the GraphQL SDL definition but certain other things are configured
```

Comment:
Let's update this slightly as we want customers to prefer the SDL. Something like this:

"Much of the behavior of the GraphQL Transform logic is configured by passing arguments to the directives in the GraphQL SDL definition, but certain things can be configured by passing parameters to the CloudFormation template itself as an escape hatch when functionality isn't in the Transformer."

Reply:
updated.